### PR TITLE
Update 'prevalence' to 'proportion of' in stat var description

### PR DESCRIPTION
**Pull request by shifucun**
"prevalence" is a very strong word match and results in non-optimal matching and ranking.

"proportion of" seems to be more general wording and works good for a range of query key words (see diff below).

I have also tried "percentage" which don't really match prevalence query very well (maybe prevalence could refer to millionth and not really used in common with percentage?)

sv diff report: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/boxu_base_uae_mem_2024_07_03_18_26_13.html

The two examples below show increased score of matching when asking in different ways.

image
image

### CODE REVIEW

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 1]
```diff
@@ -3534,28 +3535,28 @@ WHO/fullv_Male,Percentage of Full Immunization Coverage Among 1 Year Old male
...
 WHO/mslv_Female,Measles Immunization Coverage Among 1 year old female
 WHO/mslv_Male,Measles Immunization Coverage Among 1 year old male
 WHO/mslv_Rural,Measles Immunization Coverage Among 1 year old in Rural Areas
 WHO/mslv_Urban,Measles Immunization Coverage Among 1 year old in Urban areas
-WHO/NCD_BMI_18A,Prevalence Of Underweight Among Adults
-WHO/NCD_BMI_30A,Prevalence Of Obesity Among Adults
-WHO/NCD_PAC,Prevalence Of Insufficient Physical Activity Among Adults
+WHO/NCD_BMI_18A,incidence rate Of Underweight Among Adults
+WHO/NCD_BMI_30A,incidence rate Of Obesity Among Adults
+WHO/NCD_PAC,incidence rate Of Insufficient Physical Activity Among Adults
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:3546]: Couple of thoughts on "incidence rates":

1.  It feels odd to use it in the context of tobacco smoking or underweight.  It seem to make more sense to use in the context of a "disease".  In these contexts, percentage or prevalence feels more natural.

2.  Also, looking at the technical meaning of the words - https://www.nimh.nih.gov/health/statistics/what-is-prevalence#part_2624 - it seems incidence rate could imply "new cases" (probability of being diagnosed with disease) vs. prevalence which implies "existing cases in population".

Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv:3546]: updated to use "proportion of" which is generally suitable for different scenarios.

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 2]
```diff
@@ -3246,38 +3247,38 @@ Percent_Person_50To75Years_ReceivedColorectalCancerScreening,Percentage of Peopl
...
-Percent_Person_WithMentalHealthNotGood,Prevalence of poor Mental Health among adults
-Percent_Person_WithPhysicalHealthNotGood,Prevalence of poor Physical Health among adults
-Percent_Person_WithStroke,Prevalence of Stroke among adults
-dc/4lvmzr1h1ylk1,Prevalence of obesity among adult females
-dc/tcqb49dvg7k53,Prevalence of physical inactivity among adult females
-dc/nh3s4skee5483,Prevalence of diabetes among adult females
-dc/x13tvt8jsgrm4,Prevalence of obesity among adult males
-dc/ty7pq88d26963,Prevalence of physical inactivity among adult males
-dc/xmpy89x1nh8cg,Prevalence of diabetes among adult male
+Percent_Person_BingeDrinking,proportion of people with binge drinking
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:3250]: ... who binge drink?

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 3]
```diff
@@ -3233,9 +3234,9 @@ PalmerDroughtSeverityIndex_Atmosphere,Palmer Drought Severity Index
 Percent_BurnedArea_FireEvent,Percent of area that are burned in land fire
 Percent_BurnedArea_FireEvent_Forest,Percent of forest area that burned
 Percent_BurnedArea_FireEvent_Forest_HighSeverity,Percent of forest area that experienced high-severity fire
-Percent_Daily_TobaccoUsing_In_Count_Person,Prevalence of Daily Tobacco Smoking
-Percent_Daily_TobaccoUsing_In_Count_Person_Female,Prevalence of Daily Tobacco Use Among Females
-Percent_Daily_TobaccoUsing_In_Count_Person_Male,Prevalence of Daily Tobacco Use Among Mmales
+Percent_Daily_TobaccoUsing_In_Count_Person,percentage of people with daily Tobacco Smoking
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:3237]: who smoke tobacco daily

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 4]
```diff
@@ -3233,9 +3234,9 @@ PalmerDroughtSeverityIndex_Atmosphere,Palmer Drought Severity Index
 Percent_BurnedArea_FireEvent,Percent of area that are burned in land fire
 Percent_BurnedArea_FireEvent_Forest,Percent of forest area that burned
 Percent_BurnedArea_FireEvent_Forest_HighSeverity,Percent of forest area that experienced high-severity fire
-Percent_Daily_TobaccoUsing_In_Count_Person,Prevalence of Daily Tobacco Smoking
-Percent_Daily_TobaccoUsing_In_Count_Person_Female,Prevalence of Daily Tobacco Use Among Females
-Percent_Daily_TobaccoUsing_In_Count_Person_Male,Prevalence of Daily Tobacco Use Among Mmales
+Percent_Daily_TobaccoUsing_In_Count_Person,percentage of people with daily Tobacco Smoking
+Percent_Daily_TobaccoUsing_In_Count_Person_Female,percentage of Females with Daily Tobacco Use
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:3238]: who use tobacco daily

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 5]
```diff
@@ -3233,9 +3234,9 @@ PalmerDroughtSeverityIndex_Atmosphere,Palmer Drought Severity Index
 Percent_BurnedArea_FireEvent,Percent of area that are burned in land fire
 Percent_BurnedArea_FireEvent_Forest,Percent of forest area that burned
 Percent_BurnedArea_FireEvent_Forest_HighSeverity,Percent of forest area that experienced high-severity fire
-Percent_Daily_TobaccoUsing_In_Count_Person,Prevalence of Daily Tobacco Smoking
-Percent_Daily_TobaccoUsing_In_Count_Person_Female,Prevalence of Daily Tobacco Use Among Females
-Percent_Daily_TobaccoUsing_In_Count_Person_Male,Prevalence of Daily Tobacco Use Among Mmales
+Percent_Daily_TobaccoUsing_In_Count_Person,percentage of people with daily Tobacco Smoking
+Percent_Daily_TobaccoUsing_In_Count_Person_Female,percentage of Females with Daily Tobacco Use
+Percent_Daily_TobaccoUsing_In_Count_Person_Male,percentage of Males with Daily Tobacco Use
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:3239]: who use tobacco daily

**Review by pradh — APPROVED** [review]

File: tools/nl/embeddings/input/base/sheets_svs.csv (hunk 3 above)
Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv:3237]: Done

File: tools/nl/embeddings/input/base/sheets_svs.csv (hunk 4 above)
Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv:3238]: Done

File: tools/nl/embeddings/input/base/sheets_svs.csv (hunk 5 above)
Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv:3239]: Done

File: tools/nl/embeddings/input/base/sheets_svs.csv (hunk 2 above)
Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv:3250]: Done

**Pull request merged**